src/sql/src/parse.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::ast::Statement;
+
+/// Parses a raw SQL string into a [`Statement`].
```

Comment:
nit: you say `a Statement`, but this returns a `Vec`; is it always `.len()==1`?

Reply:
No, I'm just a dummy. Thanks.

Reply:
Hardly a dummy!